app/models/vm_or_template.rb

```diff
@@ -393,7 +393,7 @@ def self.task_arguments(options)
     when "remove_snapshot", "revert_to_snapshot" then
       [options[:snap_selected]]
     when "create_snapshot" then
-      [options[:name], options[:description], options[:memory]]
+      [options[:name], options[:description], options.fetch(:memory, false)]
```

Comment:
If this issue is specific to VMware, why is the solution to modify base class `VmOrTemplate`?